README.md

```diff
@@ -269,6 +273,20 @@ $http = new Server($socket, function (RequestInterface $request) {
 });
 ```
 
+As mentioned before the request implements the `ServerRequestInterface`.
+The `Server` will add server-side parameters like the remote address or cookies
+this object.
+
+```php 
+$http = new Server($socket, function (ServerRequestInterface $request) {
+    return new Response(
+        200,
+        array('Content-Type' => 'text/plain'),
+        "The remote: " . $request->getServerParams()['remote_address']
```

Comment:
Where does this name come from, are there any other params? (needs some documentation)